installing/installing_cloud_other/cloud-create-instance-other.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="cloud-create-instance-other"]
+= Creating an RHACS Cloud instance on other platforms
+include::modules/common-attributes.adoc[]
+:context: cloud-create-instance-other
+
+
+[role="_abstract"]
+Access {product-title-managed} ({product-title-managed-short}) by selecting an instance in the {cloud-console}. An *ACS instance* contains the {product-title-managed-short} management interface and services that Red Hat configures and manages for you. The management interface connects to your secured clusters, which contain the services that scan and collect information about vulnerabilities. One instance can connect to and monitor multiple clusters.
```

Comment:
Dumb question: Should this be _**the** {product-title-managed}_

Reply:
This is how it's titled in the GUI at the moment, as a menu option and a configuration item.

Reply:
(Not a dumb question!)